great_expectations/core/expectation_validation_result.py

```diff
@@ -147,15 +147,14 @@ def __ne__(self, other):
             # if invalid comparisons are attempted, the objects are not equal.
             return True
 
-    def __repr__(self):
+    def __repr__(self) -> str:
```

Comment:
Could we adjust the thresholds for type hinting?

Reply:
@cdkini Not sure what to do here -- please advise.  Thanks.

Reply:
@cdkini I think that "dunder" methods should be added to exclusion list (just like private methods) -- is it possible?  Thanks.